src/App.css

```diff
@@ -1,10 +1,12 @@
 @import url('https://fonts.googleapis.com/css2?family=Amarante&display=swap');
+@import './styles/auth.css';
 @import './styles/nav.css';
 @import './styles/sidebar.css';
 @import './styles/products.css';
 @import './styles/footer.css';
 @import './styles/variables.css';
 
```

Comment:
css files are global if you are not creating module css files so its not necessary to import all the files to app.css

Reply:
I am changing the styles of my component library and if I don't import the files the styles are not applying